app/controllers/my_facilities_controller.rb

```diff
@@ -45,12 +45,12 @@ def blood_pressure_control
                overall_patients: bp_query.overall_patients.count,
                overall_controlled_bps: bp_query.overall_controlled_bps.count}
 
-    @registered_patients_per_facility = bp_query.cohort_registrations.group(:registration_facility_id).count
-    @controlled_bps_per_facility = bp_query.cohort_controlled_bps.group(:registration_facility_id).count
-    @uncontrolled_bps_per_facility = bp_query.cohort_uncontrolled_bps.group(:registration_facility_id).count
+    @registered_patients_per_facility = bp_query.cohort_registrations_per_facility
+    @controlled_bps_per_facility = bp_query.cohort_controlled_bps_per_facility
```

Comment:
This looks nice, should we also refactor the overview query and registrations query to be consistent?